bok_choy/javascript.py

```diff
@@ -87,6 +87,7 @@ def decorator(clz):  # pylint: disable=missing-docstring
 
         # Add a `wait_for_js` method to the class
         if not hasattr(clz, 'wait_for_js'):
+            # pylint: disable= literal-used-as-attribute
```

Comment:
nit: Remove the space after the =
nit: In this case it doesn't matter because we're disabling from the last statement in the branch. Generally though I'd recommend putting the pragma on the same line (with 2 spaces before the # symbol, otherwise pep8 will complain) which will disable for only that line.